### DISCUSSION TITLE
Remove custom domain from github pages

### DISCUSSION THREAD

**Discussion by fdciabdul** [community/community#21621] [Other Feature Feedback, Questions, & Ideas]
<p>how to remove custom domain from github pages ?</p>
<p>i’m follow step from here <a href="https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site" class="inline-onebox" rel="noopener nofollow ugc">Configuring a custom domain for your GitHub Pages site - GitHub Docs</a></p>
<p>but i can’t click the remove button</p>
<p><img src="https://user-images.githubusercontent.com/31664438/181065873-35bad2d8-f326-42f0-ade4-582d9755a338.png" alt="Screenshot_2021_0510_005103" data-base62-sha1="DZy2J2OJBWe7x4WuasyX9IYWcL" width="632" height="402"></p>

**Comment by jsoref** [comment, marked as answer]
<p>Delete <a href="https://github.com/fdciabdul/fdciabdul.github.io/blob/ed2623cc0a2467d9e72e5ee7327100e0af8ab495/CNAME" class="inline-onebox" rel="noopener nofollow ugc">fdciabdul.github.io/CNAME at ed2623cc0a2467d9e72e5ee7327100e0af8ab495 · fdciabdul/fdciabdul.github.io · GitHub</a></p>
